Xamarin.Forms.Platform.iOS/Renderers/WebViewRenderer.cs

```diff
@@ -79,7 +79,8 @@ public void LoadHtml(string html, string baseUrl)
 
 		public void LoadUrl(string url)
 		{
-			LoadRequest(new NSUrlRequest(new NSUrl(url)));
+			var encodedStringUrl = new NSString(url).CreateStringByAddingPercentEscapes(NSStringEncoding.UTF8);
+			LoadRequest(new NSUrlRequest(new NSUrl(encodedStringUrl)));
```

Comment:
@rmarinho As a result of this, we now experienced double encoding of our URL´s. A developer on our team also pointed out that his method is marked as deprecated by [Apple](https://developer.apple.com/documentation/foundation/nsstring/1415058-stringbyaddingpercentescapesusin).

We are currently using IdentityModel´s RequestUrl to generate a Authorization URL, which already encode a redirect URL parameter, and as a result of this it is now double encoded.